generators/client/templates/vue/src/main/webapp/app/locale/translation.service.ts.ejs

```diff
@@ -3,18 +3,21 @@ import VueI18n from 'vue-i18n';
 
 export default class TranslationService {
 
-  public refreshTranslation(i18n: VueI18n, currentLanguage: string, newLanguage: string) {
+  public refreshTranslation(i18n: VueI18n, store: Store<{}>, newLanguage: string) {
```

Comment:
IMHO, it would be cleaner to inject store in service constructor.

In main.ts, it would be something like:
```typescript
const store = config.initVueXStore(Vue);
...
const translationService = new TranslationService(store);
...
provide: {
  ...
  translationService: () => translationService,
  ...
}
```

Reply:
I agree